policyengine_us/variables/gov/states/nm/tax/income/credits/cdcc/nm_cdcc_eligible.py

```diff
@@ -0,0 +1,47 @@
+from policyengine_us.model_api import *
+
+
+class nm_cdcc_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = (
+        "Eligible household for the New Mexico dependent child day care credit"
+    )
+    defined_for = StateCode.NM
+    unit = USD
+    definition_period = YEAR
+    reference = "https://nmonesource.com/nmos/nmsa/en/item/4340/index.do#!fragment/zoupio-_Toc140503752/BQCwhgziBcwMYgK4DsDWszIQewE4BUBTADwBdoAvbRABwEtsBaAfX2zgEYAWABgFYeAZgDsfAEwBKADTJspQhACKiQrgCe0AOSapEQmFwJlqjdt37DIAMp5SAIQ0AlAKIAZZwDUAggDkAws5SpGAARtCk7BISQA"
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        # Filer can not be a dependent on another tax return
+        dependent_on_another_return = tax_unit("dsi", period)
+        p = parameters(period).gov.states.nm.tax.income.credits.cdcc
+        # Filer has to be be gainfully employed to receive credit
+        # unless one person is disabled
+        employed = tax_unit("tax_unit_earned_income", period) > 0
```

Comment:
Employed implies a person-level variable
```suggestion
        has_earnings = tax_unit("tax_unit_earned_income", period) > 0
```